client/scss/layouts/_login.scss

```diff
@@ -1,206 +1,200 @@
-@import '../../../../../../client/scss/settings';
-@import '../../../../../../client/scss/tools';
-
 // overrides default nice padding defined in variables.scss
 $desktop-nice-padding: 100px;
 
-html,
-body,
-.content-wrapper {
+.login,
+.login .content-wrapper {
```

Comment:
Viewing this file with the "refactor" commit only and whitespace changes disabled, there aren’t that many changes: https://github.com/wagtail/wagtail/pull/8564/commits/96b584a6dba9106d368c86648c0eba9512ff5b07.